ui/css/treeherder.css

```diff
@@ -1378,104 +1169,6 @@ ul.failure-summary-list li .btn-xs {
     content: "*";
 }
 
-.btn-view-nav {
-  background-color: transparent;
-  border-color: #373d40;
-  color: lightgray;
-  border-radius: 0;
-  border-bottom: 0;
-  border-top: 0;
-  border-right: 0;
-}
-
-.btn-view-nav:hover,
-.btn-view-nav:focus,
-.btn-view-nav:active,
-.btn-view-nav.active {
-  background-color: #2c3133;
-  border-color: #1a1d20;
-  color: white;
-}
-.btn-view-nav.disabled:hover,
-.btn-view-nav.disabled:focus,
-.btn-view-nav.disabled:active,
-.btn-view-nav.disabled.active,
-.btn-view-nav[disabled]:hover,
-.btn-view-nav[disabled]:focus,
-.btn-view-nav[disabled]:active,
-.btn-view-nav[disabled].active,
-fieldset[disabled] .btn-view-nav:hover,
-fieldset[disabled] .btn-view-nav:focus,
-fieldset[disabled] .btn-view-nav:active,
-fieldset[disabled] .btn-view-nav.active {
-  color: #e0e0e0;
-  border-color: #e0e0e0;
-}
-
-.btn-view-nav-closed {
-  background-color: rgba(107, 4, 4, 0.53);
-  border-color: #22282d;
-  color: lightgray;
-  border-radius: 0;
-  border-bottom: 0;
-  border-top: 0;
-  border-left: 0;
-}
-.btn-view-nav-closed:hover,
-.btn-view-nav-closed:focus,
-.btn-view-nav-closed:active,
-.btn-view-nav-closed.active {
-  background-color: rgba(107, 4, 4, 0.53);
-  border-color: #1a1d20;
-  color: white;
-}
-.btn-view-nav-closed.disabled:hover,
-.btn-view-nav-closed.disabled:focus,
-.btn-view-nav-closed.disabled:active,
-.btn-view-nav-closed.disabled.active,
-.btn-view-nav-closed[disabled]:hover,
-.btn-view-nav-closed[disabled]:focus,
-.btn-view-nav-closed[disabled]:active,
-.btn-view-nav-closed[disabled].active,
-fieldset[disabled] .btn-view-nav-closed:hover,
-fieldset[disabled] .btn-view-nav-closed:focus,
-fieldset[disabled] .btn-view-nav-closed:active,
-fieldset[disabled] .btn-view-nav-closed.active {
-  color: #e0e0e0;
-  border-color: #e0e0e0;
-}
-
-.btn-unclassified-failures {
```

Comment:
`btn-unclassified-failures` is part of the navbar and a button that exists nowhere else, so it was moved in this PR.

![unclassifiedfailuresbtn](https://cloud.githubusercontent.com/assets/3660661/9721245/5e55c8dc-5565-11e5-9100-4102cc0e9500.jpg)

Regular resultset job buttons will move to a separate file later.